### PR TITLE
handle separately MutatingWebhookConfiguration and ValidatingWebhookConfiguration

### PR DESCRIPTION
**Pull request by savitaashture**
Issue : https://github.com/knative/serving-operator/issues/226

### CODE REVIEW

File: pkg/controller/knativeserving/knativeserving_controller.go [hunk 1]
```diff
@@ -30,6 +31,7 @@ import (
 
 const (
 	operand = "knative-serving"
+	path    = "deploy/resources/webhook/webhook.yaml"
```
Comment (jcrossley3) [inline pkg/controller/knativeserving/knativeserving_controller.go]: Name this const "webhookPath" for clarity?

**Review by jcrossley3 — COMMENTED** [review]
I'd prefer this in a `preInstall` but I'm not sure how we'd handle deletion then. Understanding that this is a temporary solution, I'm ok with it.

File: pkg/controller/knativeserving/knativeserving_controller.go (hunk 1 above)
Comment (savitaashture) [reply pkg/controller/knativeserving/knativeserving_controller.go]: Done.
Thanks

**Comment by markusthoemmes** [comment]
Obsoleted by new branch.

**Pull request closed without merging**